code/modules/vchat/js/vchat.js

```diff
@@ -578,6 +578,10 @@ function start_vue() {
 					}
 				}
 
+				newmessage.content = newmessage.content.replace(
+					/(\b(https?|ftp|file):\/\/[\-A-Z0-9+&@#\/%?=~_|!:,.;]*[\-A-Z09+&@#\/%=~_|])/img, //Honestly good luck with this regex ~Gear
```

Comment:
Seems good but allowing `file://` links seems inappropriate given that people could link people to .bat or .exe files on their own computer or do other silly things that might be undesirable in the very old version of IE that byond is using. Maybe just `https?` given that ftp is silly.

Reply:
Oop, fixed.